src/types/leads.ts

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+// import { File } from 'types/file.interface';
+// import { Location } from 'types/location.interface';
+// import { RoleInterface } from './role.interface';
+
+interface Contact {
+  value: string;
+  contacts_types_id: number;
+  weight: number;
+}
+
+interface Address {
+  address: string;
+  city: string;
+}
+
+interface CustomField {
+  data: {
+    name: string;
+    value: any;
+  }[]
+}
+
+interface People {
+  firstname: string;
+  lastname: string;
+  contacts: Contact[];
+  address: Address[];
+  custom_fields: CustomField; 
+}
+
+
+interface User {
+  id: number;
+  firstname: string;
+  lastname: string;
+}
+
+interface Owner {
+  id: number;
+  firstname: string;
+  lastname: string;
+}
+
+interface Status {
+  id: number;
+  name: string;
+  is_default: string;
+}
+interface Company {
+  id: number;
+  name: string;
+}
+interface Organization {
+  name: string;
+}
+
+export interface CreateLeadParams{
+  branch_id: number;
+  title: string;
+  pipeline_stage_id: number;
+  people: People;
+  organization: Organization;
+  custom_fields: CustomField;
+}
+
+
+export interface CreateLeadData {
+  id: number;
+  uuid: string;
+  title: string;
+  firstname: string;
+  lastname: string;
+  reason_lost: string; 
+  description: string;
+  created_at: string;
+  user: User;
+  owner: Owner;
```

Comment:
**Question:** If a user and the owner have the same attributes, why we are having 2 different interfaces?